src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@trxcllnt yeah if you inspect the `next state` object you will not get the model at all; also if you refresh the browser a couple of times you will get the error I was mentioning. The get works but not the deserialization; that's why you also have the last 404 error trying to request an [object Object] resource.